src/ContextGenerator/JsonldContextGenerator.php

```diff
@@ -29,7 +29,7 @@ class JsonldContextGenerator implements JsonldContextGeneratorInterface {
   /**
    * Constant hook alter name.
    */
-  const FIELD_TYPE_ALTER_HOOK = 'jsonld_field_mappings_alter';
+  const FIELD_TYPE_ALTER_HOOK = 'jsonld_field_mappings';
```

Comment:
ups, you missed one- Should be `const FIELD_TYPE_HOOK` ?

Reply:
Ha, yeah. Oops. I was so focused on the actual hook name that I neglected the const. Just a moment. Doing too many things at once.

Reply:
No worries, no rush, will approve and merge once that gets in. Thanks